ledger/sample/demo.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"github.com/tron-us/go-btfs-common/ledger"
+	"log"
```

Comment:
change to `github.com/tron-us/go-common/v2/log`

Reply:
fixed